{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "045c6462",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting mysql-connector-python\n",
      "  Using cached mysql_connector_python-8.0.28-py2.py3-none-any.whl (342 kB)\n",
      "Collecting protobuf>=3.0.0\n",
      "  Using cached protobuf-3.19.4-py2.py3-none-any.whl (162 kB)\n",
      "Installing collected packages: protobuf, mysql-connector-python\n",
      "Successfully installed mysql-connector-python-8.0.28 protobuf-3.19.4\n"
     ]
    }
   ],
   "source": [
    "!pip install mysql-connector-python"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "64c8b035",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "###1번째 처리: 성공\n",
      "###2번째 처리: 성공\n",
      "###3번째 처리: 성공\n",
      "###4번째 처리: 성공\n",
      "###5번째 처리: 성공\n",
      "###6번째 처리: 성공\n",
      "###7번째 처리: 성공\n",
      "###8번째 처리: 성공\n",
      "###9번째 처리: 성공\n",
      "###10번째 처리: 성공\n",
      "###11번째 처리: 성공\n",
      "###12번째 처리: 성공\n",
      "###13번째 처리: 성공\n",
      "###14번째 처리: 성공\n",
      "###15번째 처리: 성공\n",
      "###16번째 처리: 성공\n",
      "###17번째 처리: 성공\n",
      "###18번째 처리: 성공\n",
      "###19번째 처리: 성공\n",
      "###20번째 처리: 성공\n",
      "###21번째 처리: 성공\n",
      "###22번째 처리: 성공\n",
      "###23번째 처리: 성공\n",
      "###24번째 처리: 성공\n",
      "###25번째 처리: 성공\n",
      "###26번째 처리: 성공\n",
      "###27번째 처리: 성공\n",
      "###28번째 처리: 성공\n",
      "###29번째 처리: 성공\n",
      "###30번째 처리: 성공\n",
      "###31번째 처리: 성공\n",
      "###32번째 처리: 성공\n",
      "###33번째 처리: 성공\n",
      "###34번째 처리: 성공\n",
      "###35번째 처리: 성공\n",
      "###36번째 처리: 성공\n",
      "###37번째 처리: 성공\n",
      "###38번째 처리: 성공\n",
      "###39번째 처리: 성공\n",
      "###40번째 처리: 성공\n",
      "###41번째 처리: 성공\n",
      "###42번째 처리: 성공\n",
      "###43번째 처리: 성공\n",
      "###44번째 처리: 성공\n",
      "###45번째 처리: 성공\n",
      "###46번째 처리: 성공\n",
      "###47번째 처리: 성공\n",
      "###48번째 처리: 성공\n",
      "###49번째 처리: 성공\n",
      "###50번째 처리: 성공\n",
      "###51번째 처리: 성공\n",
      "###52번째 처리: 성공\n",
      "###53번째 처리: 성공\n",
      "###54번째 처리: 성공\n",
      "###55번째 처리: 성공\n",
      "###56번째 처리: 성공\n",
      "###57번째 처리: 성공\n",
      "###58번째 처리: 성공\n",
      "###59번째 처리: 성공\n",
      "###60번째 처리: 성공\n",
      "###61번째 처리: 성공\n",
      "###62번째 처리: 성공\n",
      "###63번째 처리: 성공\n",
      "###64번째 처리: 성공\n",
      "###65번째 처리: 성공\n",
      "###66번째 처리: 성공\n",
      "###67번째 처리: 성공\n",
      "###68번째 처리: 성공\n",
      "###69번째 처리: 성공\n",
      "###70번째 처리: 성공\n",
      "###71번째 처리: 성공\n",
      "###72번째 처리: 성공\n",
      "###73번째 처리: 성공\n",
      "###74번째 처리: 성공\n",
      "###75번째 처리: 성공\n",
      "###76번째 처리: 성공\n",
      "###77번째 처리: 성공\n",
      "###78번째 처리: 성공\n",
      "###79번째 처리: 성공\n",
      "###80번째 처리: 성공\n",
      "###81번째 처리: 성공\n",
      "###82번째 처리: 성공\n",
      "###83번째 처리: 성공\n",
      "###84번째 처리: 성공\n",
      "###85번째 처리: 성공\n",
      "###86번째 처리: 성공\n",
      "###87번째 처리: 성공\n",
      "###88번째 처리: 성공\n",
      "###89번째 처리: 성공\n",
      "###90번째 처리: 성공\n",
      "###91번째 처리: 성공\n",
      "###92번째 처리: 성공\n",
      "###93번째 처리: 성공\n",
      "###94번째 처리: 성공\n",
      "###95번째 처리: 성공\n",
      "###96번째 처리: 성공\n",
      "###97번째 처리: 성공\n",
      "###98번째 처리: 성공\n",
      "###99번째 처리: 성공\n",
      "###100번째 처리: 성공\n",
      "###101번째 처리: 성공\n",
      "###102번째 처리: 성공\n",
      "###103번째 처리: 성공\n",
      "###104번째 처리: 성공\n",
      "###105번째 처리: 성공\n",
      "###106번째 처리: 성공\n",
      "###107번째 처리: 성공\n",
      "###108번째 처리: 성공\n",
      "###109번째 처리: 성공\n",
      "###110번째 처리: 성공\n",
      "###111번째 처리: 성공\n",
      "###112번째 처리: 성공\n",
      "###113번째 처리: 성공\n",
      "###114번째 처리: 성공\n",
      "###115번째 처리: 성공\n",
      "###116번째 처리: 성공\n",
      "###117번째 처리: 성공\n",
      "###118번째 처리: 성공\n",
      "###119번째 처리: 성공\n",
      "###120번째 처리: 성공\n",
      "###121번째 처리: 성공\n",
      "###122번째 처리: 성공\n",
      "###123번째 처리: 성공\n",
      "###124번째 처리: 성공\n",
      "###125번째 처리: 성공\n",
      "###126번째 처리: 성공\n",
      "###127번째 처리: 성공\n",
      "###128번째 처리: 성공\n",
      "###129번째 처리: 성공\n",
      "###130번째 처리: 성공\n",
      "###131번째 처리: 성공\n",
      "###132번째 처리: 성공\n",
      "###133번째 처리: 성공\n",
      "###134번째 처리: 성공\n",
      "###135번째 처리: 성공\n",
      "###136번째 처리: 성공\n",
      "###137번째 처리: 성공\n",
      "###138번째 처리: 성공\n",
      "###139번째 처리: 성공\n",
      "###140번째 처리: 성공\n",
      "###141번째 처리: 성공\n",
      "###142번째 처리: 성공\n",
      "###143번째 처리: 성공\n",
      "###144번째 처리: 성공\n",
      "###145번째 처리: 성공\n",
      "###146번째 처리: 성공\n",
      "###147번째 처리: 성공\n",
      "###148번째 처리: 성공\n",
      "###149번째 처리: 성공\n",
      "###150번째 처리: 성공\n",
      "###151번째 처리: 성공\n",
      "###152번째 처리: 성공\n",
      "###153번째 처리: 성공\n",
      "###154번째 처리: 성공\n",
      "###155번째 처리: 성공\n",
      "###156번째 처리: 성공\n",
      "###157번째 처리: 성공\n",
      "###158번째 처리: 성공\n",
      "###159번째 처리: 성공\n",
      "###160번째 처리: 성공\n",
      "###161번째 처리: 성공\n",
      "###162번째 처리: 성공\n",
      "###163번째 처리: 성공\n",
      "###164번째 처리: 성공\n",
      "###165번째 처리: 성공\n",
      "###166번째 처리: 성공\n",
      "###167번째 처리: 성공\n",
      "###168번째 처리: 성공\n",
      "###169번째 처리: 성공\n",
      "###170번째 처리: 성공\n",
      "###171번째 처리: 성공\n",
      "###172번째 처리: 성공\n",
      "###173번째 처리: 성공\n",
      "###174번째 처리: 성공\n",
      "###175번째 처리: 성공\n",
      "###176번째 처리: 성공\n",
      "###177번째 처리: 성공\n",
      "###178번째 처리: 성공\n",
      "###179번째 처리: 성공\n",
      "###180번째 처리: 성공\n",
      "###181번째 처리: 성공\n",
      "###182번째 처리: 성공\n",
      "###183번째 처리: 성공\n",
      "###184번째 처리: 성공\n",
      "###185번째 처리: 성공\n",
      "###186번째 처리: 성공\n",
      "###187번째 처리: 성공\n",
      "###188번째 처리: 성공\n",
      "###189번째 처리: 성공\n",
      "###190번째 처리: 성공\n",
      "###191번째 처리: 성공\n",
      "###192번째 처리: 성공\n",
      "###193번째 처리: 성공\n",
      "###194번째 처리: 성공\n",
      "###195번째 처리: 성공\n",
      "###196번째 처리: 성공\n",
      "###197번째 처리: 성공\n",
      "###198번째 처리: 성공\n",
      "###199번째 처리: 성공\n",
      "###200번째 처리: 성공\n",
      "###201번째 처리: 성공\n",
      "###202번째 처리: 성공\n",
      "###203번째 처리: 성공\n",
      "###204번째 처리: 성공\n",
      "###205번째 처리: 성공\n",
      "###206번째 처리: 성공\n",
      "###207번째 처리: 성공\n",
      "###208번째 처리: 성공\n",
      "###209번째 처리: 성공\n",
      "###210번째 처리: 성공\n",
      "###211번째 처리: 성공\n",
      "###212번째 처리: 성공\n",
      "###213번째 처리: 성공\n",
      "###214번째 처리: 성공\n",
      "###215번째 처리: 성공\n",
      "###216번째 처리: 성공\n",
      "###217번째 처리: 성공\n",
      "###218번째 처리: 성공\n",
      "###219번째 처리: 성공\n",
      "###220번째 처리: 성공\n",
      "###221번째 처리: 성공\n",
      "###222번째 처리: 성공\n",
      "###223번째 처리: 성공\n",
      "###224번째 처리: 성공\n",
      "###225번째 처리: 성공\n",
      "###226번째 처리: 성공\n",
      "###227번째 처리: 성공\n",
      "###228번째 처리: 성공\n",
      "###229번째 처리: 성공\n",
      "###230번째 처리: 성공\n",
      "###231번째 처리: 성공\n",
      "###232번째 처리: 성공\n",
      "###233번째 처리: 성공\n",
      "###234번째 처리: 성공\n",
      "###235번째 처리: 성공\n",
      "###236번째 처리: 성공\n",
      "###237번째 처리: 성공\n",
      "###238번째 처리: 성공\n",
      "###239번째 처리: 성공\n",
      "###240번째 처리: 성공\n",
      "###241번째 처리: 성공\n",
      "###242번째 처리: 성공\n",
      "###243번째 처리: 성공\n",
      "###244번째 처리: 성공\n",
      "###245번째 처리: 성공\n",
      "###246번째 처리: 성공\n",
      "###247번째 처리: 성공\n",
      "###248번째 처리: 성공\n",
      "###249번째 처리: 성공\n",
      "###250번째 처리: 성공\n",
      "###251번째 처리: 성공\n",
      "###252번째 처리: 성공\n",
      "###253번째 처리: 성공\n",
      "###254번째 처리: 성공\n",
      "###255번째 처리: 성공\n",
      "###256번째 처리: 성공\n",
      "###257번째 처리: 성공\n",
      "###258번째 처리: 성공\n",
      "###259번째 처리: 성공\n",
      "###260번째 처리: 성공\n",
      "###261번째 처리: 성공\n",
      "###262번째 처리: 성공\n",
      "###263번째 처리: 성공\n",
      "###264번째 처리: 성공\n",
      "###265번째 처리: 성공\n",
      "###266번째 처리: 성공\n",
      "###267번째 처리: 성공\n",
      "###268번째 처리: 성공\n",
      "('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e01c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e07c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112a9bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1143493a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfdc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b275b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e09d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cec10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b274f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce2b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfc40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1143497f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e03a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce3a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b270a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce7c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfe20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114365940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1143496d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b94c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b9970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b271f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfe80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e51c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142802e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1143496d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b270d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b9610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1143498b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c24f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfdc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b273d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110760040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b276d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1143497f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ceef40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b276a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110760eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e07c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b239d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114627b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b275b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b271c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435e850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b9610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b94c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cffd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21e50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c68b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c29d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfc40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114627b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b277f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b275b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107dbb340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435e580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435ef40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b9610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435ec10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cea30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435ec10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1107604c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cea00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1107608b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435ec10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b236d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435e580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cee80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d9a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b270d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114627fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435e760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d1c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111180460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435e820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b94c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceb80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b270d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110760eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11435eb50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cec70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b213d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b9610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0a60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114627b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfe20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1111732b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c68b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1143496d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cec10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b235b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e52b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b233d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfe80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280e80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceaf0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfe80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349c10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d8b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e1f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cf070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110bf5400>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dfa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c28b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142809d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142805e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d0a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dd30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b9610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c26d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x112fbc040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142800a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dc70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493de50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111173190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493de80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dbe0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142802e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110bf9fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142805e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dc10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dcd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23b80>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110bf9fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110bf9ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23fd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d9a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142802b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493deb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493df40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e310>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280be0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493ea30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142803d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce2b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e1f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce1c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cec40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280a30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e00a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce6a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b233a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23d00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b277c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422310>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d6d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b274c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e02e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b232e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23ca0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dc10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111173160>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23df0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c64c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b235e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2790>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110bf9670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c2370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110760eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e02e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce490>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23fa0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e57f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23dc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23190>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce3a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0f10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110760040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5940>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e07f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cebe0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e7f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1146a3280>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b235b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cebe0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422310>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b277c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111422640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0250>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce2b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b236d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e730>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1146a3220>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceb50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493ef70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e51f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c27c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11401cfd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e07f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21cd0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce1f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6c70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e59d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5ac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6700>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1145b9970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0af0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e51f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5d90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceb20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d7f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493ebe0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5f40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e02e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c6040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114349820>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5610>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5040>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493ebe0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cea00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493d8b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23760>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cedf0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5d30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5430>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23c40>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110760eb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e05e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e59d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e8e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142807c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cec70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e910>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5b50>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cfdc0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280d30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e00d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e57f0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e670>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ce0d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107d04640>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1146a3a90>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112cef10>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493eaf0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5d30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376370>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142803d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1146a3b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147c23d0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5880>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493e850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110bf9ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1112ceee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1146a3a00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5ee0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1142802e0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280460>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0970>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ceea00>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23070>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b27e20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee7c0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0b20>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280bb0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5340>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b21580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11493dd30>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e5850>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280580>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280d60>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e03a0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cee130>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x110b23f70>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114376100>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107ceeac0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1147e0550>: Failed to establish a new connection: [Errno 61] Connection refused'))\n",
      "HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /monitor/json/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x114280520>: Failed to establish a new connection: [Errno 61] Connection refused'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "###834번째 처리: 성공\n",
      "###835번째 처리: 성공\n",
      "###836번째 처리: 성공\n",
      "###837번째 처리: 성공\n",
      "###838번째 처리: 성공\n",
      "###839번째 처리: 성공\n",
      "###840번째 처리: 성공\n",
      "###841번째 처리: 성공\n",
      "###842번째 처리: 성공\n",
      "###843번째 처리: 성공\n",
      "###844번째 처리: 성공\n",
      "###845번째 처리: 성공\n",
      "###846번째 처리: 성공\n",
      "###847번째 처리: 성공\n",
      "###848번째 처리: 성공\n",
      "###849번째 처리: 성공\n",
      "###850번째 처리: 성공\n",
      "###851번째 처리: 성공\n",
      "###852번째 처리: 성공\n",
      "###853번째 처리: 성공\n",
      "###854번째 처리: 성공\n",
      "###855번째 처리: 성공\n",
      "###856번째 처리: 성공\n",
      "###857번째 처리: 성공\n",
      "###858번째 처리: 성공\n",
      "###859번째 처리: 성공\n",
      "###860번째 처리: 성공\n",
      "###861번째 처리: 성공\n",
      "###862번째 처리: 성공\n",
      "###863번째 처리: 성공\n",
      "###864번째 처리: 성공\n",
      "###865번째 처리: 성공\n",
      "###866번째 처리: 성공\n",
      "###867번째 처리: 성공\n",
      "###868번째 처리: 성공\n",
      "###869번째 처리: 성공\n",
      "###870번째 처리: 성공\n",
      "###871번째 처리: 성공\n",
      "###872번째 처리: 성공\n",
      "###873번째 처리: 성공\n",
      "###874번째 처리: 성공\n",
      "###875번째 처리: 성공\n",
      "###876번째 처리: 성공\n",
      "###877번째 처리: 성공\n",
      "###878번째 처리: 성공\n",
      "###879번째 처리: 성공\n",
      "###880번째 처리: 성공\n",
      "###881번째 처리: 성공\n",
      "###882번째 처리: 성공\n",
      "###883번째 처리: 성공\n",
      "###884번째 처리: 성공\n",
      "###885번째 처리: 성공\n",
      "###886번째 처리: 성공\n",
      "###887번째 처리: 성공\n",
      "###888번째 처리: 성공\n",
      "###889번째 처리: 성공\n",
      "###890번째 처리: 성공\n",
      "###891번째 처리: 성공\n",
      "###892번째 처리: 성공\n",
      "###893번째 처리: 성공\n",
      "###894번째 처리: 성공\n",
      "###895번째 처리: 성공\n",
      "###896번째 처리: 성공\n",
      "###897번째 처리: 성공\n",
      "###898번째 처리: 성공\n",
      "###899번째 처리: 성공\n",
      "###900번째 처리: 성공\n",
      "###901번째 처리: 성공\n",
      "###902번째 처리: 성공\n",
      "###903번째 처리: 성공\n",
      "###904번째 처리: 성공\n",
      "###905번째 처리: 성공\n",
      "###906번째 처리: 성공\n",
      "###907번째 처리: 성공\n",
      "###908번째 처리: 성공\n",
      "###909번째 처리: 성공\n",
      "###910번째 처리: 성공\n",
      "###911번째 처리: 성공\n",
      "###912번째 처리: 성공\n",
      "###913번째 처리: 성공\n",
      "###914번째 처리: 성공\n",
      "###915번째 처리: 성공\n",
      "###916번째 처리: 성공\n",
      "###917번째 처리: 성공\n",
      "###918번째 처리: 성공\n",
      "###919번째 처리: 성공\n",
      "###920번째 처리: 성공\n",
      "###921번째 처리: 성공\n",
      "###922번째 처리: 성공\n",
      "###923번째 처리: 성공\n",
      "###924번째 처리: 성공\n",
      "###925번째 처리: 성공\n",
      "###926번째 처리: 성공\n",
      "###927번째 처리: 성공\n",
      "###928번째 처리: 성공\n",
      "###929번째 처리: 성공\n",
      "###930번째 처리: 성공\n",
      "###931번째 처리: 성공\n",
      "###932번째 처리: 성공\n",
      "###933번째 처리: 성공\n",
      "###934번째 처리: 성공\n",
      "###935번째 처리: 성공\n",
      "###936번째 처리: 성공\n",
      "###937번째 처리: 성공\n",
      "###938번째 처리: 성공\n",
      "###939번째 처리: 성공\n",
      "###940번째 처리: 성공\n",
      "###941번째 처리: 성공\n",
      "###942번째 처리: 성공\n",
      "###943번째 처리: 성공\n",
      "###944번째 처리: 성공\n",
      "###945번째 처리: 성공\n",
      "###946번째 처리: 성공\n",
      "###947번째 처리: 성공\n",
      "###948번째 처리: 성공\n",
      "###949번째 처리: 성공\n",
      "###950번째 처리: 성공\n",
      "###951번째 처리: 성공\n",
      "###952번째 처리: 성공\n",
      "###953번째 처리: 성공\n",
      "###954번째 처리: 성공\n",
      "###955번째 처리: 성공\n",
      "###956번째 처리: 성공\n",
      "###957번째 처리: 성공\n",
      "###958번째 처리: 성공\n",
      "###959번째 처리: 성공\n",
      "###960번째 처리: 성공\n",
      "###961번째 처리: 성공\n",
      "###962번째 처리: 성공\n",
      "###963번째 처리: 성공\n",
      "###964번째 처리: 성공\n",
      "###965번째 처리: 성공\n",
      "###966번째 처리: 성공\n",
      "###967번째 처리: 성공\n",
      "###968번째 처리: 성공\n",
      "###969번째 처리: 성공\n",
      "###970번째 처리: 성공\n",
      "###971번째 처리: 성공\n",
      "###972번째 처리: 성공\n",
      "###973번째 처리: 성공\n",
      "###974번째 처리: 성공\n",
      "###975번째 처리: 성공\n",
      "###976번째 처리: 성공\n",
      "###977번째 처리: 성공\n",
      "###978번째 처리: 성공\n",
      "###979번째 처리: 성공\n",
      "###980번째 처리: 성공\n",
      "###981번째 처리: 성공\n",
      "###982번째 처리: 성공\n",
      "###983번째 처리: 성공\n",
      "###984번째 처리: 성공\n",
      "###985번째 처리: 성공\n",
      "###986번째 처리: 성공\n",
      "###987번째 처리: 성공\n",
      "###988번째 처리: 성공\n",
      "###989번째 처리: 성공\n",
      "###990번째 처리: 성공\n",
      "###991번째 처리: 성공\n",
      "###992번째 처리: 성공\n",
      "###993번째 처리: 성공\n",
      "###994번째 처리: 성공\n",
      "###995번째 처리: 성공\n",
      "###996번째 처리: 성공\n",
      "###997번째 처리: 성공\n",
      "###998번째 처리: 성공\n",
      "###999번째 처리: 성공\n",
      "###1000번째 처리: 성공\n",
      "###1001번째 처리: 성공\n",
      "###1002번째 처리: 성공\n",
      "###1003번째 처리: 성공\n",
      "###1004번째 처리: 성공\n",
      "###1005번째 처리: 성공\n",
      "###1006번째 처리: 성공\n",
      "###1007번째 처리: 성공\n",
      "###1008번째 처리: 성공\n",
      "###1009번째 처리: 성공\n"
     ]
    }
   ],
   "source": [
    "import mysql.connector\n",
    "import requests, json\n",
    "import time\n",
    "\n",
    "# 데이터베이스 컨넥션을 생성한다.\n",
    "mydb = mysql.connector.connect(\n",
    "    host=\"127.0.0.1\",\n",
    "    user=\"smartnqi\",\n",
    "    passwd=\"nwai1234!\",\n",
    "    database=\"smart\"\n",
    ")\n",
    "\n",
    "# 커서를 생성한다.\n",
    "cur = mydb.cursor()\n",
    "\n",
    "# 데이터를 조회할 쿼리문을 작성한다. \n",
    "sql = '''\n",
    "    select \n",
    "        'call' as dataType,\n",
    "        phone_no,\n",
    "        meastime,\n",
    "        networkId,\n",
    "        groupId,\n",
    "        currentTime,\n",
    "        timeline,\n",
    "        cellId,\n",
    "        currentCount,\n",
    "        ispId,\n",
    "        testNetworkType,\n",
    "        userInfo1,\n",
    "        userInfo2,\n",
    "        siDo,\n",
    "        guGun,\n",
    "        addressDetail,\n",
    "        udpJitter,\n",
    "        downloadBandwidth,\n",
    "        uploadBandwidth,\n",
    "        sinr,\n",
    "        isWifi,\n",
    "        latitude,\n",
    "        longitude,\n",
    "        bandType,\n",
    "        p_dl_earfcn,\n",
    "        p_pci,\n",
    "        p_rsrp,\n",
    "        p_SINR,\n",
    "        NR_EARFCN,\n",
    "        NR_PCI,\n",
    "        NR_RSRP,\n",
    "        NR_SINR\n",
    "        from tb_ndm_data_measure  \n",
    "        where \n",
    "            meastime between 20211101000000000 and 20211101235959999\n",
    "         --  and phone_no in (1029213866, 1029213948)\n",
    "        -- and userInfo1 = '경상남도-사천시-남양동'\n",
    "       \n",
    "        order by savetime\n",
    "    '''\n",
    "\n",
    "# 쿼리문을 실행한다.\n",
    "cur.execute(sql)\n",
    "row_headers=[x[0] for x in cur.description]\n",
    "res = cur.fetchall()\n",
    "\n",
    "# 데이터를 가져와서 JSON 데이터를 생성한다.\n",
    "for idx, row in enumerate(res):\n",
    "    tmpData = dict(zip(row_headers,row))\n",
    "    jsonData = json.dumps(tmpData)\n",
    "    try:\n",
    "        r = requests.post(\"http://localhost:8000/monitor/json/\", data=jsonData)\n",
    "        # 전송실패 시 오류값을 출력한다.\n",
    "        if r.status_code == 200:\n",
    "            print(f\"###{idx+1}번째 처리: 성공\")\n",
    "        else:\n",
    "            print(f\"###{idx+1}번째 처리: 오류{r.status_code}{r.text}\")\n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(str(e))\n",
    "\n",
    "    # 초당 3개씩 전달하도록 한다. \n",
    "    # 너무 다량의 데이터를 연속해서 보내면 서버단에서 부하로 오류가 발생한다. \n",
    "#     if idx % 3 == 0:\n",
    "#         time.sleep(1)\n",
    "                  "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ebdee317",
   "metadata": {
    "collapsed": true
   },
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "de440360",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
